app/Console/RemoveLabelOffer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex\Console;
+
+use CultuurNet\UDB3\Offer\Commands\RemoveLabel;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RemoveLabelOffer extends AbstractRemoveLabel
```

Comment:
```suggestion
final class RemoveLabelFromOffer extends AbstractRemoveLabel
```
(Make sure to update usages if you apply this)